apps/browser/qml/pages/components/TabView.qml

```diff
@@ -11,26 +11,18 @@
 
 import QtQuick 2.1
 import Sailfish.Silica 1.0
+import Sailfish.Silica.private 1.0 as Private
+import Sailfish.Browser 1.0
 import org.nemomobile.configuration 1.0
 import "." as Browser
+import "../../shared" as Shared
 
-SilicaGridView {
-    id: tabView
+Private.TabView {
+    id: tabs
```

Comment:
There's an error from Overlay.qml attempting to access `Browser.TabView.positionViewAtIndex()` It doesn't look like it's being caused by your changes here, but do you think it would be possible to fix it along with these changes?
```
[W] unknown:656 - file:///usr/share/sailfish-browser/pages/components/Overlay.qml:656: ReferenceError: positionViewAtIndex is not defined
[W] unknown:665 - file:///usr/share/sailfish-browser/pages/components/Overlay.qml:665: TypeError: Property 'positionViewAtIndex' of object TabView_QMLTYPE_144
```

Reply:
Applied the hack, as there are no ideas how to do it correctly 
```
        Component.onCompleted:  {
            if ((index / columns * cellHeight) > tabView.height) {
                tabView.loaded = true
            }
        }
```